### DISCUSSION TITLE
[gatsby-plugin-image] Is there a way to specify in GatsbyImage for images to be generated by height rather then width?

### DISCUSSION THREAD

**Discussion by jamborski-dev** [gatsbyjs/gatsby#34620] [Help]
Hi! I am building a video/photo portfolio for a friend and following the design example he gave - once a gallery item is openned the full widnow size slider showes up. 

In my implementation of it I am using Prismic + Gatsby v4 + pure-react-carousel + new GatsbyImage plugin.

Everything works up until I resize the window from a desktop view to a mobile view. Becauase the sizes of the generated images are being calculated from a width breakpoints as soon as the image that is naturally a landscape format goes below 800px of width it starts too loose quality as much as its almost blurred image width in a portrait view. The slider container where the GatsbyImage container is spaninng on entire height of the viewport upscalling image which is too small for it to be a HQ image.

![example-gatsby-image-wide](https://user-images.githubusercontent.com/46531778/151661746-15b0baac-a095-4016-ac61-61a40109e948.png)

![example-gatsby-image-narrow](https://user-images.githubusercontent.com/46531778/151661752-36f2776a-6e7e-4d7f-a289-3701ecd7858d.png)

Is there a way to overide this behaviour somehow? I've tried to set defaults for breakpoints, formats, quality but with no luck. 

Thanks in advance!

**Comment by Isaac-Tait** [comment]
This looks a lot like an issue I have been having for a while now https://github.com/gatsbyjs/gatsby/issues/31775#issuecomment-1022268793
